postgres-cmd-spec.json

```diff
@@ -0,0 +1,29 @@
+{
+  "label": "postgres command",
+  "environment": [
+    "PATH=/usr/lib64/ccache:/usr/bin:/usr/sbin:/bin:/sifive/tools/rust/stable/1.72.0/bin",
+    "TERM=xterm-256color",
+    "DATA_DIR=/tmp/pg_data",
+    "PG_DIR=/scratch/jakee/pg-server2"
+  ],
+  "user-id": 0,
+  "group-id": 0,
+  "visible": [],
+  "directory": ".",
+  "stdin": "",
+  "resources": [],
+  "isolate-network": true,
+  "isolate-pids": true,
+  "mount-ops": [
+    {"type": "tmpfs", "destination": "/tmp"},
+    {"type": "tmpfs", "destination": "/root"}
```

Comment:
I don't think I need this anymore

Reply:
It turns out that I do in fact need an extra temp home directory somewhere because some tools like ccache will add things to it.